diagnostic.md

```diff
@@ -45,6 +74,10 @@ weather === "sunny"
 
 What do these expressions evaluate to?  Explain your answers.
 
+the weather variable is assigned the string 'sunny' in line 1.
+Then in line 2,  the value is true because 3 equals means "two strings are strictly equal when they have the same sequence of characters, same length, and same characters in corresponding positions."
+
+Resource: Stack overflow for === and running in node.
```

Comment:
`let weather` evaluates to undefined

`weather = 'sunny'` evaluates to 'sunny'

`weather === 'sunny'` evaluates to true